charts/ndb-operator/templates/v1_service_controller-manager-metrics-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    control-plane: controller-manager
```

Comment:
let's use {{ include "ndb-operator.fullname" . }}- as prefix of every label value